tests/Doctrine/Tests/DBAL/Functional/ResultCacheTest.php

```diff
@@ -10,7 +10,14 @@
  */
 class ResultCacheTest extends \Doctrine\Tests\DbalFunctionalTestCase
 {
+    /**
+     * @var int[][]|string[][]
+     */
     private $expectedResult = array(array('test_int' => 100, 'test_string' => 'foo'), array('test_int' => 200, 'test_string' => 'bar'), array('test_int' => 300, 'test_string' => 'baz'));
+
+    /**
+     * @var \Doctrine\DBAL\Logging\DebugStack
```

Comment:
Please move fully qualified names to the `use` section and use aliases.

Reply:
Shouldn't we add [this](https://github.com/Symplify/CodingStandard#types-should-not-be-referenced-via-a-fullypartially-qualified-name-but-via-a-use-statement-wrench) to our CS as well?

Reply:
Looks like we do have the check for class names in the code but it doesn't work for comments. Inspecting:
```php
<?php

$conn = new \Doctrine\DBAL\Connection();
```
will produce:
```
------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
------------------------------------------------------------------------------------------------
 3 | ERROR | [x] Class \Doctrine\DBAL\Connection should not be referenced via a fully qualified
   |       |     name, but via a use statement.
------------------------------------------------------------------------------------------------
```

Reply:
@morozov Can you create brief repro code & report to Slevomat repo? It may've simply been overlooked since Slevomat guys afaik prefer FQCN in params/typehints. :)

Reply:
@Majkl578 please see https://github.com/slevomat/coding-standard/issues/258.

Reply:
Ready to test: https://github.com/slevomat/coding-standard/commit/c73280903ece590687d0d9abdddb8e8efdb017ca